apps/projects/test/projects.test.js

```diff
@@ -89,72 +100,316 @@ contract('Projects App', accounts => {
       root,
       { from: root }
     )
-    await app.initialize({ from: accounts[0] })
+
+    // Deploy test Bounties contract
+    registry = await StandardBounties.new(web3.toBigNumber(owner1))
+    bounties = StandardBounties.at(registry.address)
+
+    await app.initialize(registry.address)
   })
 
-  context('creating and retrieving repos and bounties', function() {
-    it('creates a repo id entry', async function() {
-      repoID = (await app.addRepo(
-        'MDEyOk9yZ2FuaXphdGlvbjM0MDE4MzU5',
-        'MDEwOlJlcG9zaXRvcnkxMTYyNzE4MDk='
-      )).logs.filter(x => x.event == 'RepoAdded')[0].args.id
+  context('creating and retrieving repos and bounties', () => {
+    let repoId
+
+    beforeEach(async () => {
+      repoId = addedRepo(
+        await app.addRepo(
+          'MDEyOk9yZ2FuaXphdGlvbjM0MDE4MzU5',
+          'MDEwOlJlcG9zaXRvcnkxMTYyNzE4MDk='
+        )
+      )
+    })
+
+    it('creates a repo id entry', async () => {
       assert.equal(
-        repoID,
+        repoId,
         '0xd1f2b806d3ffc90a501a7c22dbbcbb3f1f14e136cd9da208dc0b6e6b0f64b777',
         'repo is created and hashed ID is returned'
       )
     })
 
-    it('retrieves repo information successfully', async function() {
-      const owner1 = accounts[0]
-      repoID = (await app.addRepo(
-        'MDEyOk9yZ2FuaXphdGlvbjM0MDE4MzU5',
-        'MDEwOlJlcG9zaXRvcnkxMTYyNzE4MDk='
-      )).logs.filter(x => x.event == 'RepoAdded')[0].args.id
-      repoInfo = await app.getRepo(repoID, { from: owner1 })
-      result = web3.toAscii(repoInfo[0])
+    it('retrieve repo array length', async () => {
+      const repolength = await app.getRepoArrayLength()
+      assert(repolength, 2, 'valid repo length returned')
+    })
+
+    it('retrieve repo information successfully', async () => {
+      const repoInfo = await app.getRepo(repoId, { from: owner1 })
+      const result = web3.toAscii(repoInfo[0])
 
-      // result = repoInfo[0]
       assert.equal(
         result,
         'MDEyOk9yZ2FuaXphdGlvbjM0MDE4MzU5',
-        'invalid repo info returned'
+        'valid repo info returned'
       )
     })
 
-    it('accepts bounties for issues in an added repo', async function() {
-      const bountyAdder = accounts[2]
-      repoID = (await app.addRepo('abc', String(123))).logs.filter(
-        x => x.event == 'RepoAdded'
-      )[0].args.id
-      issue3Receipt = (await app.addBounties(repoID, [1, 2, 3], [10, 20, 30], {
-        from: bountyAdder,
-      })).logs.filter(x => x.event == 'BountyAdded')[2]
-      issue3Bounty = issue3Receipt.args.bountySize.toNumber()
-      assert.equal(issue3Bounty, 30, 'bounty not added')
+    context('standard bounty verification tests', () => {
```

Comment:
If we're going to do standard bounties tests they should be in another test file. Only exception would be if it's tests that are routed through the Projects contract.